snowplow_tracker/emitters.py

```diff
@@ -83,7 +83,9 @@ def __init__(
         :param batch_size:  The maximum number of queued events before the buffer is flushed. Default is 10.
         :type  batch_size:  int | None
         :param on_success:  Callback executed after every HTTP request in a flush has status code 200
-                            Gets passed the number of events flushed.
+                            Gets passed one argument:
+                            1) If method is "post": The sent data in string form;
```

Comment:
Isn't it the other way? The post method is passed a list of payload dictionaries and the get method is passed the request data as string? Also if this is the case, we might need to update the type hint for the callback as currently it only covers the list of dictionaries.